stringify.js

```diff
@@ -0,0 +1,24 @@
+const stringify = (data, depth = 5) => {
+  return data === undefined
+    ? 'null'
```

Comment:
`'undefined'` would be more appropriate

Reply:
JSON doesnt support `undefined`, but does support `null` and Im not sure we need a string of `'undefined'` 🤔